cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -586,7 +586,7 @@ def _json_dict_(self) -> dict[str, Any]:
         return cirq.obj_to_dict_helper(self, ["component_gates"])
 
     @classmethod
-    def _from_json_dict_(cls, component_gates: list[cirq.Gate], **kwargs: Any) -> ParallelGates:
+    def _from_json_dict_(cls, component_gates: list[cirq.Gate], **_kwargs: Any) -> ParallelGates:
```

Comment:
predates this pr, but while we're here i think this should just be
```suggestion
    def _from_json_dict_(cls, component_gates: list[cirq.Gate], **_kwargs: object) -> ParallelGates:
```